docker/README.md

```diff
@@ -88,8 +88,31 @@ directory as the dockerfile is in:
 docker build -t asreview:1.0rc0 .
 ```
 
+![instruction image](./img/dockerfile.png)
```

Comment:
I think the added value of this image is low.

Reply:
I agree 👍🏻